### PR TITLE
Support for PHP 8.1

### LINKED ISSUE #40: PHP 8.1 support
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes

#### Summary

PHP with [version 8.1 is released](https://www.php.net/releases/8.1/index.php) and for this component the support of the new version is still missing.

### PR DESCRIPTION
**Pull request by snapshotpl**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | yes
| RFC           | no
| QA            | yes

### CODE REVIEW

**Comment by snapshotpl** [comment]
Any idea why on PHP 8.0 composer skip platform requirements, but on 8.1 is not?

File: composer.json [hunk 1]
```diff
@@ -24,28 +24,28 @@
...
         "symfony/dom-crawler": "^5.4 || ^6.0"
     },
     "require-dev": {
-        "laminas/laminas-coding-standard": "~2.2.1",
+        "laminas/laminas-coding-standard": "~2.3.0",
         "laminas/laminas-i18n": "^2.6",
         "laminas/laminas-log": "^2.7.1",
         "laminas/laminas-modulemanager": "^2.7.1",
         "laminas/laminas-mvc-console": "^1.1.8",
         "laminas/laminas-mvc-plugin-flashmessenger": "^1.0",
```
Comment (froschdesign) [inline composer.json]: See: https://github.com/laminas/laminas-mvc-plugin-flashmessenger/releases/tag/1.4.0

**Review by Koen1999 — APPROVED** [review]
Looks good to me!

File: composer.json [hunk 2]
```diff
@@ -24,28 +24,28 @@
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
         "laminas/laminas-console": "^2.6",
```
Comment (froschdesign) [inline composer.json:28]: If laminas-console is still required, how should an end user install laminas-test?

Comment (snapshotpl) [reply composer.json:28]: `--ingnore-platform-req` ? Same like PHP 8.0 is "supported" now. It's definitely last version of 3.x, 4.x without `laminas-console` is waiting for release.

Comment (froschdesign) [reply composer.json:28]: > Same like PHP 8.0 is "supported" now. 

Unbeautiful!

> …4.x without laminas-console is waiting for release.

Are you planning a release right after 3.8.0?

Comment (snapshotpl) [reply composer.json:28]: Yes!

**Review by froschdesign — APPROVED** [review]

**Review by Ocramius — APPROVED** [review]
LGTM, thanks @snapshotpl!

**Pull request merged**